src/__snapshots__/trackerHoc.test.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`trackerHoc should render component with trackedPromiseInProgress equals false and area equals "default" when render promiseTrackerHoc 1`] = `
```

Comment:
Are useful these snaphsots?

Reply:
Sure, it's the way to know the spec result. Instead of you have to create somethink like we did with karma + enzyme:

```javascript
...
// Assert
expect(component.html()).toEqual(`
<TestSpinnerComponent
  area="default"
>
...
`)
```
If there are any changes in component implementation, we will see changes on this snapshot too